src/index.html

```diff
@@ -1,12 +1,323 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="styles/index.scss"
+    />
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="logo"
+      >
```

Comment:
Fix it everywhere if you have only 2 attributes
```suggestion
      <a href="#home" class="logo">
```